vote/error.go

```diff
@@ -93,13 +93,19 @@ type messageError struct {
 }
 
 // MessageError creates an typed error with a message.
-func MessageError(t TypeError, format string, a ...any) error {
```

Comment:
Why integrating code changes into a go update PR?
Can you please split the go update and the changes in error messages?

Reply:
I think the answer here is the same as in https://github.com/OpenSlides/openslides-autoupdate-service/pull/1147